R/testHelpers.R

```diff
@@ -122,8 +126,8 @@ run_servr <- function(directory = ".", port = 4848,
                       code = "servr::httd(dir='%s', port=%d)") {
   dir <- normalizePath(directory, winslash = "/", mustWork = TRUE)
   cmd <- sprintf(
-    paste("library(methods); write.table(Sys.getpid(), file='%s', append=T, row.name=F, col.names=F);", code),
-    pid_file(), dir, port
+    paste("write.table(Sys.getpid(), file='%s', append=T, row.name=F, col.names=F);", code),
```

Comment:
Is there a good reason for not loading __methods__? I think it's safer to leave it in there.